pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +273,18 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) isDockerAPIVersionCompatible(input string) bool {
```

Comment:
This function name doesn't describe what sort of compatibility it's checking for. I would call this `dockerSupportsParallelPulls` or something like that.